src/python/pants/backend/python/lint/ruff/subsystem.py

```diff
@@ -36,6 +43,12 @@ def opt_out(cls, tgt: Target) -> bool:
         return tgt.get(SkipRuffField).value
 
 
+class RuffMode(str, Enum):
+    FIX = "fix"
+    FORMAT = "format"
```

Comment:
I think it would make sense to align these with the goals, so:
```suggestion
    FORMAT = "fmt"
```

Reply:
Done!